---
layout: talk_reveal3
title: Forecasting the Future of Flu
venue: Chicago, IL
event: Options IX
author: Trevor Bedford
draft: true
---

<!--

Outline:
* Vaccine updates and antigenic drift
  - Drifted clades quickly take over
  - Spindly tree and rapid clade growth indicates adaptive evolution
  - Rapid surveillence and analysis necessary
* nextflu
* Antigenic cartography and tree-based antigenic analysis
  - Model validation
  - Phylogenetic interpolation of titer and identification of missing clades
* Fitness models and forecasting
* How to identify high fitness viruses
  - Sequence mutations
  - Clade behavior
  - Antigenic evolution
* Projections and model validation
  - When does the forecast fail? Emerging clades.
* New mutations difficult
  - Interpolation vs extrapolation of genotype-phenotype
* The world today and projections
  - "Barring substantial changes in other clades, we predict the (HA1:171K, HA2:77V/155E) variant to dominate."
* Future improvements
  - Extend to other seasonal viruses
  - Extend to NA
  - Better phenotype: integration of neutralization, egg adaptation
  - Geographic origins of antigenic variants
* Looking past seasonal influenza
  - Ebola / Zika
  - Avian flu / swine flu
  - General surveillence tool

-->

<!-- Color ramp: ["#5097BA", "#5BA6A7", "#69B091", "#79B77D", "#8BBB6A", "#9EBE5A", "#B2BD4D", "#C4B945", "#D4B13F", "#DFA43B"] -->

<section data-background="#000000" data-background-iframe="https://earth.nullschool.net/#current/wind/surface/level/orthographic=-112.41,39.56,1182">
	<h2 class="title">{{ page.title }}</h2>
	<br>
	<p class="title">
	Trevor Bedford (@trvrb)
	<br>
	{{ page.date | date: "%-d %b %Y" }}
	<br>
	{{ page.event }}
	<br>
	{{ page.venue }}	
</section>

<section data-background="#000000" data-background-iframe="https://www.wunderground.com/cgi-bin/findweather/getForecast?query=chicago">
</section>

<section>
	<h3>What would it look like to have forecast for flu?</h3>
	<iframe class="stretch" data-src="https://www.wunderground.com/cgi-bin/findweather/getForecast?query=chicago"></iframe>
</section>

<section>
	<h3>Influenza H3N2 vaccine updates</h3>
	<img class="stretch" src="/images/talks/flu_vaccine_updates.png">
</section>

<section>
	<h3>H3N2 phylogeny showing antigenic drift</h3>
	<img class="stretch" src="/images/talks/flu_h3n2_tree.png">				
</section>

<section>
	<h3>H3N2 phylogeny showing antigenic drift</h3>
	<img class="stretch" src="/images/talks/flu_h3n2_tree_vaccines.png">				
</section>

<section data-background="#559EB1">
	<h3 class="title">Drift variants rapidly take over the virus population</h3>
	<br>
	<h3 class="title">Timely surveillance and rapid analysis essential to understand ongoing influenza evolution</h3>
</section>

<section>
	<h2>
		<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span>
	</h2>
	<p>Project to provide a real-time view of the evolving influenza population
	<p class="fragment" data-fragment-index="1">All in collaboration with <a href="https://neherlab.wordpress.com/">Richard Neher</a></p>
	<img class="stretch fragment" data-fragment-index="1" src="/images/talks/neher.jpg">
</section>

<section>
	<h3>
		<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span>
		pipeline
	</h3>
	<p>
	<ol>
		<li>Download all recent HA sequences from GISAID</li>
		<li>Filter to remove outliers</li>		
		<li>Subsample across time and space</li>
		<li>Align sequences</li>
		<li>Build tree</li>
		<li>Estimate frequencies</li>
		<li>Infer antigenic phenotypes</li>
		<li>Export for visualization</li>
	</ol>
</section>

<section>
	<h3>Up-to-date analysis publicly available at:</h3>
	<h2>
		<a href="http://nextflu.org">
			<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span><span class="text-gray">.org</span>
		</a>
	</h2>
</section>

<section data-background="#79B77D">
	<h2 class="title">Antigenic evolution</h2>
</section>

<section>
	<h3>Influenza hemagglutination inhibition (HI) assay</h3>
	<img class="stretch" src="/images/talks/hi_assay.png">				
</section>

<section>
	<h3>HI measures cross-reactivity across viruses</h3>
	<img class="stretch" src="/images/talks/hi_cross_reactivity.jpg">				
</section>

<section>
	<h3>Data in the form of table of maximum inhibitory titers</h3>
	<img class="stretch" src="/images/talks/hi_table.png">
	<div class="citation">
		<a href="http://www.crick.ac.uk/research/worldwide-influenza-centre/annual-and-interim-reports/">NIMR 2010</a>
	</div>
</section>

<section>
	<h3>Model HI titers by estimating branch effects</h3>
	<img class="stretch" src="/images/talks/titer_model_with_table.png">
	<div class="citation">
		<a href="http://bedford.io/papers/neher-titers/">Neher et al 2016</a>
	</div>	
</section>

<section>
	<h3>Model is highly predictive of missing titer values</h3>
	<img class="stretch" src="/images/talks/titers_validation.png">
	<div class="citation">
		<a href="http://bedford.io/papers/neher-titers/">Neher et al 2016</a>
	</div>		
</section>


<section>
	<h3>Broad patterns agree with cartographic analyses</h3>
	<img class="stretch" src="/images/talks/titers_h3n2_drift_comparison.png">
</section>


<section>
	<h3>Recent HI data from WHO CC London annual and interim reports</h3>
</section>

<section>
	<h3>Up-to-date analysis at:</h3>
	<h2>
		<a href="http://nextflu.org">
			<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span><span class="text-gray">.org</span>
		</a>
	</h2>
</section>

<section data-background="#69B091">
	<h3 class="title">Analytics can identify drifted clades (with drops in HI titers and signature genetic changes)</h3>
	<br>
	<h3 class="title">The frequencies of these clades can then be tracked through time</h3>
</section>

<section>
	<h3>Clades continually appear and sometimes fix</h3>
	<img class="stretch" src="/images/talks/flu_h3n2_tree.png">				
</section>

<section>
	<h3>Clade frequency trajectories over past 15 years</h3>
	<img class="stretch" src="/images/talks/prediction_combined_trajectories.png">
</section>

<section>
	<h3>Interference visible between clades</h3>
	<br>
	<div style="vertical-align: bottom; display: table-cell;">
		<img width="49%" src="/images/talks/prediction_combined_trajectories.png">
		&nbsp;
		<img width="49%" src="/images/talks/clonal_interferance.jpg">
	</div>	
</section>

<section>
	<h3>Chance of fixation based on initial frequency</h3>
	<img class="stretch" src="/images/talks/prediction_chance_of_fixation.png">
</section>

<section>
	<h3>Chance of fixation based on initial frequency and growth rate</h3>
	<img class="stretch" src="/images/talks/prediction_chance_of_fixation_by_growth_rate.png">
</section>

<section data-background="#8BBB6A">
	<h2 class="title">Forecasting</h2>
</section>

<section>
	<h3><p><q>The future is here, it's just not evenly distributed yet</q> &mdash; William Gibson</p></h3>
</section>

<section>
	<h3>USA music industry, 2011 dollars per capita</h3>
	<img class="stretch" src="/images/talks/music_industry.jpg">
</section>

<section>
	<h3>Influenza population turnover</h3>
	<img class="stretch" src="/images/talks/flu_turnover.png">
</section>

<section>
	<h3>Vaccine strain selection timeline</h3>
	<img class="stretch" src="/images/talks/flu_vaccine_schedule.jpg">
</section>

<section>
	<h3>Seek to explain change in clade frequencies over 1 year</h3>
	<img class="stretch" src="/images/talks/prediction_clade_frequencies.png">
</section>

<section>
	<h3>Fitness models can project clade frequencies</h3>
	<br>
	<p>Clade frequencies $X$ derive from the fitnesses $f$ and frequencies $x$ of constituent viruses, such that</p>
	<p>$$\hat{X}_v(t+\Delta t) = \sum_{i:v} x_i(t) \, \mathrm{exp}(f_i \, \Delta t)$$</p>
	<p>This captures clonal interference between competing lineages</p>
	<img class="stretch" src="/images/talks/clonal_interferance.jpg">		
	<div class="citation">
		<a href="http://www.nature.com/nature/journal/v507/n7490/full/nature13087.html">&#321;uksza and L&auml;ssig 2014</a>
	</div>	
</section>

<section>
	<h3>Predictive fitness models</h3>
	<br>
	<p>A simple predictive model estimates the fitness $f$ of virus $i$ as</p>
	<p>$$\hat{f}_i = \beta^\mathrm{ep} \, f_i^\mathrm{ep} + \beta^\mathrm{ne} \, f_i^\mathrm{ne}$$</p>
	<p>where $f_i^\mathrm{ep}$ measures cross-immunity via substitutions at epitope sites and $f_i^\mathrm{ep}$ measures mutational load via substitutions at non-epitope sites</p>
	<img class="stretch" src="/images/talks/prediction_ll.jpg">	
	<div class="citation">
		<a href="http://www.nature.com/nature/journal/v507/n7490/full/nature13087.html">&#321;uksza and L&auml;ssig 2014</a>
	</div>	
</section>

<section>
	<h3>We implement a similar model based on two predictors</h3>
	<br>
	<ol>
		<li class="larger">Clade frequency change</li>
		<li class="larger">Antigenic advancement</li>
	</ol>
</section>

<section>
	<h3>Project frequencies forward, <br> growing clades have high fitness</h3>
	<img class="stretch" src="/images/talks/prediction_frequency_trajectories.png">
</section>

<section>
	<h3>Calculate HI drop from ancestor, <br> drifted clades have high fitness</h3>
	<img class="stretch" src="/images/talks/prediction_antigenic_drift.png">
</section>

<section>
	<h3>Fitness model parameterization</h3>
	<br>
	<p>Our predictive model estimates the fitness $f$ of virus $i$ as</p>
	<br>
	<p>$$\hat{f}_i = \beta^\mathrm{freq} \, f_i^\mathrm{freq} + \beta^\mathrm{HI} \, f_i^\mathrm{HI}$$</p>
	<br>	
	<p>We learn coefficients and validate model based on previous 15 H3N2 seasons</p>
</section>	


<section>
	<h3>Validation of 1 year clade frequency change</h3>
	<img class="stretch" src="/images/talks/prediction_validation_trajectories.png">
</section>


<section>
	<h3>Clade growth rate is well correlated (<i>&rho;</i> = 0.66)</h3>
	<img class="stretch" src="/images/talks/prediction_validation_growth_rates.png">
</section>

<section>
	<h3>Growth vs decline correct in 84% of cases</h3>
	<img class="stretch" src="/images/talks/prediction_validation_growth_rates_window.png">
</section>


<section>
	<h3>Clade error increases steadily over time</h3>
	<img class="stretch" src="/images/talks/prediction_trajectories_envelope.png">
</section>


<section>
	<h3>Trajectories show more detailed congruence</h3>
	<img class="stretch" src="/images/talks/prediction_trajectories_grid.png">
</section>

<section>
	<h3>Formalizes intuition about drivers of influenza dynamics</h3>
	<br>
	<table>
		<thead>
			<tr class="smaller">
				<th width="24%">Model</th>
				<th width="19%">Ep coefficient</th>
				<th width="19%">HI coefficient</th>
				<th width="19%">Freq error</th>
				<th width="19%">Growth corr</th>
			</tr>
		</thead>
		<tbody>
			<tr>
				<td>Epitope only</td>
				<td>2.36</td>
				<td>--</td>
				<td>0.10</td>
				<td>0.57</td>
			</tr>
			<tr>
				<td>HI only</td>
				<td>--</td>
				<td>2.05</td>
				<td>0.08</td>
				<td>0.63</td>
			</tr>
			<tr>
				<td>Epitope + HI</td>
				<td>-0.11</td>
				<td>2.15</td>
				<td>0.08</td>
				<td>0.67</td>
			</tr>						
		</tbody>
	</table>
</section>

<section>
	<h3>Further work on predictive modeling</h3>
	<br>
	<ol>	
		<li>Integrate data predictors and data sources, e.g. geography</li>
		<li>Possible to build predictive models for H1N1 and B and to forecast NA evolution</li>
	</ol>
</section>

<section>
	<h3>Phylogeny of H3 with geographic history</h3>
	<img class="stretch" src="/images/talks/pathogen_phylogeny_flu.png">
	<div class="citation">
		<a href="http://bedford.io/papers/bedford-global-circulation/">Bedford et al 2015</a>
	</div>
</section>


<section>
	<h3>Infer geographic transition matrix</h3>
	<img class="stretch" src="/images/talks/flu_geo_h3_map.png">
</section>

<section data-background="/images/talks/flight_network.jpg">
</section>

<section>
	<h3>Air travel predicts migration rates</h3>
	<img class="stretch" src="/images/talks/flu_geo_predictors.png">
	<div class="citation">
		<a href="http://bedford.io/papers/lemey-air-travel/">
			Lemey et al 2014
		</a>
	</div>	
</section>


<section>
	<h3>Geographic location of phylogeny trunk</h3>
	<img class="stretch" src="/images/talks/flu_geo_h3_trunk.png">
	<div class="citation">
		<a href="http://bedford.io/papers/bedford-global-circulation/">Bedford et al 2015</a>
	</div>	
</section>

<section>
	<h3>Real-time analyses are <i>actionable</i> and thus, may inform influenza vaccine strain selection</h3>
</section>

<section>
	<h3>Extending to Ebola, Zika and other things at:</h3>
	<h2>
		<a href="http://nextstrain.org/">
			<span class="text-gray">nextstrain.org/</span>
		</a>
	</h2>	
</section>

<section>
	<h3>Acknowledgements</h3>
	<br>
	<p class="smaller">WHO Global Influenza Surveillance Network, Worldwide Influenza Centre at the Francis Crick Institute, Richard Neher, Colin Russell, Andrew Rambaut</p>
	<div class="pull-right" style="vertical-align: bottom; display: table-cell;">
		<img width="50%" src="/images/talks/logo_fred_hutch.png">
		&nbsp;
		<img width="25%" src="/images/talks/logo_midas.jpg">	
	</div>
</section>

<section data-background="#000000">
	<h3 class="title">Contact</h3>
	<br>
	<p>
	<ul class="title">
		<li>Website: bedford.io</li>
		<li>Twitter: @trvrb</li>
		<li>Slides: bedford.io{{ page.url }}</li>
	</ul>
</section>
